File: BankApplicationAPI/BankApplicationAPI/Models/ComplaintResolution.cs
namespace BankApplicationAPI.Models;

public partial class ComplaintResolution
{
    public int ResolutionId { get; set; }

    public int? ComplaintId { get; set; }

    public DateTime? ResolutionDate { get; set; }

    public string? ResolutionMethod { get; set; }

    public string? ResolutionDescription { get; set; }

    public string? EmployeeId { get; set; }

    public virtual Complaint? Complaint { get; set; }

    public virtual Employee? Employee { get; set; }
}

File: BankApp/src/app/Modules/ComplaintResolution.ts
import { Complaint } from "./Complaint";
import { Employee } from "./Employee";

export interface ComplaintResolution {
    resolutionId?: number | null;
    complaintId?: number | null;
    resolutionDate?: Date | null;
    resolutionMethod?: string | null;
    resolutionDescription?: string | null;
    employeeId?: string | null;
    complaint?: Complaint | null;
    employee?: Employee | null;
}